loggroup-lambda-connector/loggroup-lambda-cft.json

```diff
@@ -1,5 +1,22 @@
 {
     "AWSTemplateFormatVersion": "2010-09-09",
+    "Parameters": {
+        "LambdaARN": {
+            "Type" : "String",
+            "Default": "arn:aws:lambda:us-east-1:123456789000:function:TestLambda",
+            "Description": "Enter ARN for target lambda function"
+        },
+        "LogGroupPattern": {
+            "Type" : "String",
+            "Default": "Test",
+            "Description": "Enter regex for matching logGroups"
+        },
+        "UseExistingLogs": {
+            "Type" : "String",
+            "Default": "false",
+            "Description": "Enter true for subscribing existing logs"
```

Comment:
changed to "Enter: true (all lower case) for subscribing existing logs" or ignore casing and trim in the code to catch different values by users.